zulipterminal/api_types.py

```diff
@@ -21,8 +21,8 @@ class PrivateComposition(TypedDict):
 class StreamComposition(TypedDict):
     type: Literal["stream"]
     content: str
-    to: str  # stream name  # TODO: Migrate to using int (stream id)
-    subject: str  # TODO: Migrate to using topic
+    to: Optional[int]
```

Comment:
This is a separate migration. It may be OK in this PR, but certainly in another commit.